corehq/apps/domain/templates/domain/billing_statements.html

```diff
@@ -174,8 +215,20 @@
             {% endblocktrans %}
         </h3>
     </script>
+
+    <script type="text/html" id="bulk-payment-method-modal-title">
+        <h3>
+            {% trans 'Bulk Payment' %}
+        </h3>
+    </script>
 {% endblock %}
 
 {% block modals %}{{ block.super }}
-    {% include 'domain/partials/payment_modal.html' with payment_modal_id="paymentModal" title_template="payment-method-modal-title" cost_item_template="cost-item-template" %}
+    {% with process_invoice_payment_url as process_payment_url %}
+        {% include 'domain/partials/payment_modal.html' with payment_modal_id="paymentModal" title_template="payment-method-modal-title" cost_item_template="cost-item-template" %}
+    {% endwith %}
+    {% with process_bulk_payment_url as process_payment_url %}
```

Comment:
I'm actually kind of surprised this worked. The payment modal hard codes an id into the form `payment-modal` and then the handler grabs that id to submit the form. If there are multiple modals on the page, then this isn't valid html